proposals/nnnn-TouchCoord-outside-video-screen-range.md

```diff
@@ -0,0 +1,99 @@
+# TouchCoord outside video screen range
+
+* Proposal: [SDL-XXXX](nnnn-TouchCoord-outside-video-screen-range.md)
+* Author: [Masato Ogawa](https://github.com/masatoogawa)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / RPC]
+
+## Introduction
+
+Updates `TouchCoord` for gesture recognition across the SDL Video screen and the Native screen
+
+## Motivation
+
+We have use cases for keeping gesture recognition from the SDL Video screen to the Native screen. Since the current `TouchCoord` does not support negative values, the SDL applications can not keep to recognize some gestures for upward or leftward direction.
```

Comment:
Please update second sentence to "Since the current `TouchCoord` does not support negative values, SDL applications cannot recognize some gestures for upward or leftward direction."

Reply:
Thanks. The sentence is updated.